dev/benchmarks/macrobenchmarks/test_driver/picture_cache_complexity_scoring_perf_test.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:macrobenchmarks/common.dart';
+
+import 'util.dart';
+
+void main() {
+  macroPerfTest(
+    'picture_cache_complexity_scoring_perf',
+    kPictureCacheComplexityScoringRouteName,
+    pageDelay: const Duration(seconds: 1),
+    driverOps: (FlutterDriver driver) async {
+      final SerializableFinder tabBarView = find.byValueKey('tabbar_view');
```

Comment:
Should this have a unique name so as not to be confused with the tbv used in the picture_cache benchmark you copied this from?

Reply:
You changed the Key, but not the Finder...?

Reply:
I fixed it, accidentally edited it in the wrong file...